docs/vcf_to_bigquery.md

```diff
@@ -0,0 +1,134 @@
+# VCF to BigQuery
+
+The VCF to BigQuery pipeline performs the main application of Variant Transforms:
+parsing input VCF files and loading them into BigQuery tables. We perform this task
+through 3 intermediate stages:
+
+![VCF to BigQuery stages](images/vcf_to_bq_stages.svg)
+
+## Stage1: Run Dataflow pipeline
+
+In this stage, Variant Transforms reads VCF files from the input
+[Google Cloud Storage](https://cloud.google.com/storage/docs/how-to)
+bucket, [shards](sharding.md) the variants, and writes the output [Avro](http://avro.apache.org/)
+files back to Google Cloud Storage. The following image shows an example of 
+this Dataflow job:
+
+![VCF to BigQuery Dataflow pipeline](images/vcf_to_bq_dataflow_pipeline.png)
```

Comment:
Based this figure `ReadingFromVcf` will take 23 days, and then `VariantToAvro` will take another 22-26 days?  So you would expect folks to wait 2 months to process their data!?!?  I thought Google Cloud Storage with Dataflow was supposed to be super-fast and a serious workhorse.  I think there requires quick a significantly huge re-optimization on this pipeline.

Reply:
Wall time unit is vCPU hours:
>Approximate time spent in this step on initializing, processing data, shuffling data, and terminating, across all threads in all workers. For composite steps, the sum of time spent in the component steps. This estimate helps you identify slow steps.

For more information please refer to [Dataflow documentations](https://cloud.google.com/dataflow/)

Reply:
Yes, I understand the difference between wall-clock time and CPU time, and I get what vCPU means in terms of sharing bare-metal CPUs among virtual machines.  The key is that users come to Google expecting things to go much faster via Cloud-optimized infrastructure and resources, and these numbers would not give them confidence.